_ml-commons-plugin/gpu-acceleration.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: default
+title: GPU acceleration
+parent: Model-serving framework
+nav_order: 150
+---
+
+# GPU acceleration 
+
+When running a natural language processing (NLP) model in your OpenSearch cluster with a machine learning (ML) node, you can achieve better performance on the ML node using graphic processing unit (GPU) acceleration. GPUs can work in tandem with the central processing unit (CPU) of your cluster to speed up the model upload and training. 
+
+## Supported GPUs
+
+Currently, ML nodes support any CUDA-enabled GPUs, including
+
+- [NVIDIA G-series instances](https://aws.amazon.com/nvidia/)
+- [AWS Inferentia](https://aws.amazon.com/machine-learning/inferentia/)
+
+If you need GPU power, you can provision GPU instances through [Amazon EC2](https://aws.amazon.com/ec2/). For more information on how to provision a GPU instance, see [Recommended GPU Instances](https://docs.aws.amazon.com/dlami/latest/devguide/gpu.html).
```

Comment:
```suggestion
If you need GPU power, you can provision GPU instances through Amazon Elastic Compute Cloud ([Amazon EC2])(https://aws.amazon.com/ec2/). For more information on how to provision a GPU instance, see [Recommended GPU Instances](https://docs.aws.amazon.com/dlami/latest/devguide/gpu.html).
```

Reply:
I'm not certain if I added the parentheses in the right place, but Amazon EC2 needs to be defined on first appearance.